tests/unit_tests/sqlalchemy_data_models/conftest.py

```diff
@@ -0,0 +1,85 @@
+"""The base conftest.py for the unit tests.
+
+This file was generated on July 30, 2020
+"""
+__author__ = "Gage Larsen"
+__copyright__ = "Copyright: (c) Aquaveo 2020"
+__maintainer__ = "Gage Larsen"
+__email__ = "glarsen@aquaveo.com"
+
+
+import uuid
+
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+
+from param_persist.sqlalchemy.models import Base, InstanceModel, ParamModel
+
+
+@pytest.fixture(scope='session')
+def engine():
```

Comment:
You need to teach me this fixture voodoo magic. Are these injected into the test functions with arguments by the same name?

Reply:
i believe that is how it works